test/integration/requests/accountChannels.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable mocha/no-hooks-for-single-case -- Use of hooks is restricted when there is a single test case. */
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { AccountChannelsRequest } from '../../../src'
```

Comment:
```suggestion
import { AccountChannelsRequest } from 'xrpl-local'
```
All imports should be from `xrpl-local` instead of from `../../../src`, because otherwise browser tests will fail